osf/models/collection.py

```diff
@@ -181,8 +181,18 @@ def save(self, *args, **kwargs):
 
         if first_save:
             # Set defaults for M2M
-            self.collected_types = ContentType.objects.filter(app_label='osf', model__in=['abstractnode', 'collection', 'preprint'])
-            # Set up initial permissions
+            self.collected_types = ContentType.objects.filter(
+                app_label='osf',
+                model__in=['abstractnode', 'collection', 'preprint']
+            )
+            content_type = ContentType.objects.filter(
```

Comment:
Out of curiosity, what changed that caused this to be added in?

Reply:
just a copy and paste error